src/lib/components/event/event-date-filter.svelte

```diff
@@ -83,44 +107,22 @@
         </div>
       </div>
     {/each}
-
     <div class="option pr-4">
       <div class="check" />
       <div class="my-2 w-full border-b-2 border-gray-300 pr-2" />
     </div>
-  {/if}
-  {#each dateOptions as { label, option } (option)}
     <div
       class="option"
-      class:active={$timeFormat === option}
-      on:click={() => onDateOptionClick(option)}
-      data-cy="event-date-filter-{option}"
+      class:active={$eventShowElapsed === 'true'}
```

Comment:
Why isn't `$eventShowElapsed` a boolean?

Reply:
Cuz persistStore -> localStorage -> strings. But we probably create a wrapper for booleans

Reply:
Ahh ok makes sense